frontend/javascripts/oxalis/model/sagas/save_saga.js

```diff
@@ -100,17 +101,39 @@ import window, { alert, document, location } from "libs/window";
 
 import { enforceSkeletonTracing } from "../accessors/skeletontracing_accessor";
 
-const byteArrayToLz4Array = createWorker(compressLz4Block);
+const _byteArrayToLz4Array = createWorker(compressLz4Block);
+const decompressToTypedArray = (
+  bucket: DataBucket,
+  compressedData: Uint8Array,
+): ?BucketDataArray => {
+  const decompressedBackendData = _byteArrayToLz4Array(compressedData, false);
+  if (decompressedBackendData == null) {
+    // Can this happen?
```

Comment:
@MichaelBuessemeyer Do you know whether lz4 decompression can return null? Old code seemed to handle this for some reason? See https://github.com/scalableminds/webknossos/pull/5955/files#diff-acc1d929544bdde3687f0e79f303c4f23b61ca97fafe5e4579202972d2e37acaL413

Reply:
Nope, I don't know. 
According to the [docs](https://www.npmjs.com/package/lz4js) the compression and decompression always return an array and therefore not null.
I could only guess that this maybe is a flow thing because we compress / decompress using a worker with comlink 🤷 . Maybe the worker and comlink usage introduce the option that it might be null.

So I think as long as flow does not what a if-"catch" for a null we can leave it out.